src/packages/@ncigdc/components/Modals/FirstTimeModal.js

```diff
@@ -29,8 +29,9 @@ const FirstTimeModal = () => (
         GDC Data Release Notes
       </ExternalLink>
       {" "}
-      prior to accessing this web site as the Release Notes provide details about data updates, known issues and workarounds.
+      prior to accessing this web site as the Release Notes provide details about data updates, and known issues and workarounds.
```

Comment:
or if `known issues and workarounds` are two things I don't think you need the extra `and`